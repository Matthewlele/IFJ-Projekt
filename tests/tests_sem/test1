def foo? (arg1)
  x = arg1
  print y
  arg1 = arg1 * 5
  x = arg1
end

param1 = 5
print foo? param
