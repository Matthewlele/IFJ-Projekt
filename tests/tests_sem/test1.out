expected SEMANTIC ERR: line 3
